### ISSUE TITLE
1.1.0 is not backwards compatible with 1.0.0 due to import change

### ISSUE THREAD

**Issue by BlooJeans** [mreinstein/remove-array-items#7]
Heads up for people: version 1.0.0 -> 1.1.0 changed how this library exports (not 100% backwards compatible, breaking SemVer). Depending on how the requiring library does its imports (esm vs cjs), it might break your library/app if it was requiring this library via "^1.0.0"

In particular, for nanobus
remove-array-items v1.0.0
```
var splice = require('remove-array-items')
// splice is the Function
```

remove-array-items v1.1.0
```
var splice = require('remove-array-items')
// splice is {default: Function, __esModule: true}
```

Examples of issues:
https://github.com/choojs/nanobus/issues/37
https://github.com/TypeStrong/ts-loader/issues/874

**Comment by mreinstein** [comment]
@BlooJeans good catch! Can you send a fix PR? I'll push it as `1.1.1` to fix the semver breakage.

**Comment by mreinstein** [comment]
What bundler/build process is this breaking for? This should work on rollup/webpack 2+ as-is.

**Comment by mreinstein** [comment]
fixed via https://github.com/mreinstein/remove-array-items/commit/66b0bb72b858809345c93d20bb5a2a2763f645de . waiting for the tests to pass on ci, and I'll publish this as 1.1.1 on npm.

**Comment by BlooJeans** [comment]
I'm not overly familiar with webpack/rollup etc, so I'm not too sure what piece there is acting strangely.  Testing the nanobus in isolation is actually fine, however clearly something with the build process handles 1.0.0 and 1.1.0 differently.

Apparently the best practice is to publish this fix as 1.2.0 instead of 1.1.1, and then you can publish 1.1.0 as 2.0.0
(from https://github.com/semver/semver/blob/master/semver.md)

> What do I do if I accidentally release a backwards incompatible change as a minor version?
> As soon as you realize that you've broken the Semantic Versioning spec, fix the problem and release a new minor version that corrects the problem and restores backwards compatibility. Even under this circumstance, it is unacceptable to modify versioned releases. If it's appropriate, document the offending version and inform your users of the problem so that they are aware of the offending version.
> 

Thanks for hopping on this so quickly!

**Comment by mreinstein** [comment]
I published as a patch version so that more people would get this updated module (most people use `^` but some still use `1.1.x`)

> Thanks for hopping on this so quickly!

Thanks for bringing it to my attention! Yay open source! :)
